content/coordination/onboarding/index.en.md

```diff
@@ -143,29 +143,29 @@ we can add them to our records."
 
 ## Appendix B
 
-> confirm list of organisers - email template
+> confirm list of organizers - email template
 
 ```markdown
 Hi \<\<first name\>\>,
 
-Thanks for sending through the updated list of organisers. I sent the
-slack invites to the new organisers.
+Thanks for sending through the updated list of organizers. I sent the
+slack invites to the new organizers.
 
-We keep information about chapters and organisers on this list on
+We keep information about chapters and organizers on this list on
 GitHub:
 https://github.com/rladies/starter-kit/blob/master/Current-Chapters.csv.
 
--   Could you please send name and surname of the new organisers so that
+-   Could you please send name and surname of the new organizers so that
     I can update the chapter list?
 
 -   Could you also please confirm whether I should remove the following
-    from the list of organisers?
+    from the list of organizers?
 
     -   \<\<NAME1\>\>
 
     -   \<\<NAME2\>\>
 
-We assume former organisers provide all the necessary information to new
+We assume former organizers provide all the necessary information to new
 organisers before passing the baton :)
```

Comment:
"organisers" is consistently used within this document, and is correct UK spelling. I suggest we dont change that.